### ISSUE TITLE
Format changes every save

### ISSUE THREAD

**Issue by Dulanic** [matthewpi/vscode-caddyfile-support#13]
**System Information**
* Operating System: Mix, VS code in Windows, connected via SSH to linux.
* Extension Version: 0.1.1
* VSCode Version: 1.55.2
* `caddy` Version: v2.4.0-beta.2 h1:DUaK4qtL3T0/gAm0fVVkHgcMN04r4zGpfPUZWHRR8QU=

**Describe the bug**
Every time I save/format the indenting flips from what it was before. see this, all I was doing is ctrl+s.

https://dulanic.com/img/GoldenrodPomeranian_.mp4

**Steps to Reproduce**
1. Format thje same Caddyfile twice.

**Expected Behavior**
The format stays the same.

**Screenshots**
https://dulanic.com/img/GoldenrodPomeranian_.mp4

**Additional Context**
Caddyfile /w limited section causing it:
```

{
	default_sni dulanic.com
	acme_ca https://acme-v02.api.letsencrypt.org/directory
	email {$ACMEEMAIL}
	#debug
}
(proxyheaders) {
	flush_interval -1
	header_up X-Real-IP {http.request.header.CF-Connecting-IP}
	header_up X-Forwarded-For {http.request.header.CF-Connecting-IP}
	header_up X-Forwarded-Host {hostport}
}

```

**Comment by matthewpi** [comment]
The formatting functionality is provided by the `caddy` executable through the use of the `fmt` command.

Formatting works perfectly fine with `v2.2.0 h1:sMUFqTbVIRlmA8NkFnNt9l7s0e+0gw+7GPIrhty905A=`, however it breaks with `v2.4.0-beta.2 h1:DUaK4qtL3T0/gAm0fVVkHgcMN04r4zGpfPUZWHRR8QU=`.

Working versions:
- `v2.2.0 h1:sMUFqTbVIRlmA8NkFnNt9l7s0e+0gw+7GPIrhty905A=`
- `v2.2.1 h1:Q62GWHMtztnvyRU+KPOpw6fNfeCD3SkwH7SfT1Tgt2c=`

Broken versions:
- `v2.3.0 h1:fnrqJLa3G5vfxcxmOH/+kJOcunPLhSBnjgIvjXV/QTA=`
- `v2.4.0-beta.2 h1:DUaK4qtL3T0/gAm0fVVkHgcMN04r4zGpfPUZWHRR8QU=`
- `v2.4.0-rc.1 h1:tZl6bDhlwtRwuWpebRUYpDJPhJaGyrXIMp7fmuMXwMc=`

**Comment by matthewpi** [comment]
> What's broken about it?

https://streamable.com/ouziak

After some testing https://github.com/caddyserver/caddy/commit/635f075f187ac73c65e6939b8402494469a4d627 seems to be the culprit which makes sense after looking at the tests that were added.

**Comment by francislavoie** [comment]
FYI @mholt, we should probably fix that before v2.4.0, shouldn't be difficult. ☝️

**Comment by francislavoie** [comment]
A PR would be great ❤️ we're planning to tag v2.4.0 probably today so sooner the better! 😁

**Comment by francislavoie** [comment]
Oh, the problem is with the env var actually, not the comment.

**Comment by matthewpi** [comment]
> Oh, the problem is with the env var actually, not the comment... or rather the combination of the two.

Yeah, that's what I am trying to figure out right now.  Without the comment it works fine but with both the environment variable and comment it messes up the rest of the file.

**Comment by matthewpi** [comment]
This should be resolved by caddy `v2.4.0` (once it releases).

**Comment by mholt** [comment]
Releasing now
